security/pkg/k8s/tokenreview/k8sauthn_test.go

```diff
@@ -120,7 +120,7 @@ func TestOnMockAPIServer(t *testing.T) {
 		authn := NewK8sSvcAcctAuthn(s.httpServer.URL+"/"+tc.cliConfig.reviewPath, tc.cliConfig.tlsCert,
 			tc.cliConfig.reviewerToken)
 
-		_, err := authn.ValidateK8sJwt(tc.cliConfig.jwt, jwt.JWTPolicyThirdPartyJWT)
+		_, err := authn.ValidateK8sJwt(tc.cliConfig.jwt, jwt.JWTPolicyThirdPartyJWT, "kubernetes")
```

Comment:
Why the clusterID here in the ValidateK8sJwt() is "kubernetes"? Using "kubernetes" as an ID is confusing and hard to understand. IMO, a cluserID is something like "cluster1", "cluster2", etc.

Reply:
I am using kubernetes as default value in testing to align with what we have in runtime:
```- name: ISTIO_META_CLUSTER_ID
  value: "{{ .Values.global.multiCluster.clusterName | default `Kubernetes` }}"```

Reply:
That is what we currently have all over in multi cluster docs.  If there is a desire to change it, we need to have a separate issue to track, but let's not do it in this PR.